command/cat.go

```diff
@@ -0,0 +1,90 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/peak/s5cmd/storage"
+	"github.com/peak/s5cmd/storage/url"
+)
+
+var catHelpTemplate = `Name:
+	{{.HelpName}} - {{.Usage}}
+
+Usage:
+	{{.HelpName}} [options] source destination
+
+Options:
+	{{range .VisibleFlags}}{{.}}
+	{{end}}
+Examples:
+	1. Print an S3 object's content to stdout
```

Comment:
we don't have a check if given argument has an s3 scheme.

Reply:
replaced it with 'remote'

Reply:
I mean, if we only allow `cat` to print remote object content, we need a check at `Before` stage.

Reply:
Updated, also added bucket check